app/views/devise/registrations/edit.html.haml

```diff
@@ -1,15 +1,15 @@
-%h2
+%h2.m-4.text-black.text-center
   Edit #{resource_name.to_s.humanize}
 = form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
   = devise_error_messages!
-  .field
+  .field.block.text-black.m-2.font-bold.md:text-center.mb-1.md:mb-0.pr-4
```

Comment:
I want you to move these classes to the css file. In the html file you leave the `.field` and move the rest to the application.css, it should look like this
```
.field {
  @apply block text-black m-2 font-bold pr-4 mb-1;
}

@screen md {
  .field {
    @apply text-center mb-0;
  }
} 
```